src/nanorc/logbook.py

```diff
@@ -66,55 +64,26 @@ def _start_new_message_thread(self):
         user = self.session_handler.nanorc_user.username
-
-        elisa_arg = copy.deepcopy(self.elisa_arguments)
-
-        elisa_user = credentials.get_login('elisa')
-
-        import tempfile
-        with tempfile.NamedTemporaryFile() as tf:
-            try:
-                sso = {"ssocookie": self.session_handler.generate_elisa_cern_cookie(self.website, tf.name)}
-                elisa_arg.update(sso)
-                elisa_inst = Elisa(**elisa_arg)
-                answer = None
-                if not self.current_id:
-                    self.log.info("ELisA logbook: Creating a new message thread")
-                    message = MessageInsert()
-                    message.author = user
-                    message.subject = subject
-                    for attr_name, attr_data in self.message_attributes[command].items():
-                        if attr_data['set_on_new_thread']:
-                            setattr(message, attr_name, attr_data['value'])
-                    message.systemsAffected = ["DAQ"]
-                    message.body = body
-                    answer = elisa_inst.insertMessage(message)
-
-                else:
-                    self.log.info(f"ELisA logbook: Answering to message ID{self.current_id}")
-                    message = MessageReply(self.current_id)
-                    message.author = user
-                    message.systemsAffected = ["DAQ"]
-                    for attr_name, attr_data in self.message_attributes[command].items():
-                        if attr_data['set_on_reply']:
-                            setattr(message, attr_name, attr_data['value'])
-                    message.body = body
-                    answer = elisa_inst.replyToMessage(message)
-                self.current_id = answer.id
-
-            except ElisaError as ex:
-                self.log.error(f"ELisA logbook: {str(ex)}")
-                self.log.error(answer)
-                raise ex
-
-            except Exception as e:
-                self.log.error(f'Exception thrown while inserting data in elisa:')
-                self.log.error(e)
-                import logging
+        #Since NAnoRC is sending these messages, is the system always DAQ?
+        data = {'author':user, 'title':subject, 'body':body, 'command':command, 'systems':["DAQ"]}
+
+        if not self.current_id:
+            r = requests.post(f'{self.socket}/v1/elisaLogbook/new_message', auth=(fooUsr,barPass), data=data)
+        else:
+            data["id"=self.current_id]
+            r = requests.put(f'{self.socket}/v1/elisaLogbook/reply_to_message', auth=(fooUsr,barPass), data=data)
+
+        response = r.json()
+        if r.status_code != 201:
+            self.log.error(f'Exception thrown while inserting data in elisa:')
+            e = (response['response'])
+            self.log.error(e)
+            import logging
                 if logging.DEBUG >= logging.root.level:
```

Comment:
Indentation error

Reply:
I guess mentally I was treating the import like an if, fixed